src/krishna/Assignment_15_16/NestedSwitchExmp.java

```diff
@@ -0,0 +1,171 @@
+package krishna.Assignment_15_16;
+import java.util.Scanner;
+/*
+ A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+ 
+ */
+public class NestedSwitchExmp {
+	int iciciBalance = 5000, hdfcBalance =6000, bobBalance = 4000, sbiBalance =8000;
+	static int totalBalance=0;
+	static int totalDebitCount,totalCreditCount;
+	static int iciciCreditCount,hdfcCreditCount,bobCreditCount,sbiCreditCount;
+	static int iciciDebitCount,hdfcDebitCount,bobDebitCount,sbitDebitCount;
+	
+	void operation(String bank,String ops, int amount) {
+		switch (ops) {
+			case "DEBIT":
+				switch (bank) {
+					case "HDFC":
+						if (hdfcBalance>amount) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)